packages/lib/services/plugins/utils/manifestFromObject.ts

```diff
@@ -49,6 +49,7 @@ export default function manifestFromObject(o: any): PluginManifest {
 		permissions: permissions,
 
 		_recommended: getBoolean('_recommended', false, false),
+		_built_in: getBoolean('_built_in', false, false),
```

Comment:
Is that used somewhere?

Reply:
Not currently. But I thought maybe we can use it in the future. For example, if we decided to have a special badge for default plugins. But if you want, I will remove it.